Torch 2.3.0 (next ver) fails with AttributeError: 'Parameter' object has no attribute 'fget'

Torch CPU tests are failing with Torch 2.3.0 (Not yet in PyPi) which is downloaded since we pull wheel files from PyTorch downloads folder directly.
Pin Torch to < 2.3.0 to resolve this issue.

Log: https://github.com/keras-team/keras/actions/runs/8804994492/job/24166626153
```
keras/src/layers/core/dense.py:150: in call
    x = ops.matmul(inputs, self.kernel)
keras/src/ops/numpy.py:3582: in matmul
    if any_symbolic_tensors((x1, x2)):
keras/src/ops/numpy.py:3584: in torch_dynamo_resume_in_matmul_at_3582
    return backend.numpy.matmul(x1, x2)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

x1 = tensor([[1., 1., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1..., 1., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1., 1., 1.]])
x2 = <KerasVariable shape=(4, 3), dtype=float32, path=example_model/kernel>

>   def matmul(x1, x2):
E   AttributeError: 'Parameter' object has no attribute 'fget'

keras/src/backend/torch/numpy.py:60: AttributeError
________________________ TestTrainer.test_fit_flow_jit _________________________
```

--- a/requirements-jax-cuda.txt
+++ b/requirements-jax-cuda.txt
@@ -3,7 +3,7 @@ tensorflow-cpu~=2.16.1  # Pin to TF 2.16
 
 # Torch cpu-only version (needed for testing).
 --extra-index-url https://download.pytorch.org/whl/cpu
-torch>=2.1.0
+torch>=2.1.0, <2.3.0
 torchvision>=0.16.0
 
 # Jax with cuda support.

--- a/requirements-tensorflow-cuda.txt
+++ b/requirements-tensorflow-cuda.txt
@@ -3,7 +3,7 @@ tensorflow[and-cuda]~=2.16.1  # Pin to TF 2.16
 
 # Torch cpu-only version (needed for testing).
 --extra-index-url https://download.pytorch.org/whl/cpu
-torch>=2.1.0
+torch>=2.1.0, <2.3.0
 torchvision>=0.16.0
 
 # Jax cpu-only version (needed for testing).

--- a/requirements.txt
+++ b/requirements.txt
@@ -2,8 +2,9 @@
 tensorflow-cpu~=2.16.1  # Pin to TF 2.16
 
 # Torch.
+# TODO: Pin to < 2.3.0 (GitHub issue #19602)
 --extra-index-url https://download.pytorch.org/whl/cpu
-torch>=2.1.0
+torch>=2.1.0, <2.3.0
 torchvision>=0.16.0
 
 # Jax.

